mmaction/models/heads/bbox_head.py

```diff
@@ -172,7 +177,12 @@ def loss(self,
             labels = labels[pos_inds]
 
             bce_loss = F.binary_cross_entropy_with_logits
-            losses['loss_action_cls'] = bce_loss(cls_score, labels)
+
+            loss = bce_loss(cls_score, labels, reduction='none')
+            pt = torch.exp(-loss)
+            F_loss = self.focal_alpha * (1 - pt)**self.focal_gamma * loss
+            losses['loss_action_cls'] = torch.mean(F_loss)
```

Comment:
How about moving this part as `class FocalLoss` and using it by build_loss(xxx)?

Reply:
like BaseHead

Reply:
I think we can make FocalLoss a separate module when there is evidence that it can lead to performance gain on other tasks. For now, it is only used in the training of AVA.